content/02.results.md

```diff
@@ -62,6 +60,48 @@ Full details including exclusion thresholds are available in supplementary.
 
 Summary of site coverage post QC exclusions.
 
+### Species assignment and sex calling
+
+The Anopheles gambiae complex is a crypic group of sibling species, with no single locus offering unambiguous resolution of species.
```

Comment:
Going to need some good refs here - probably something like (Scott, Fanello, Santolamazza, but see Vincente, 'some cryptic species papers', Ag1000G Consortium papers). e.g. Yes, there are lots of markers that folk claim are good, but when we use them to look widely across Africa, they're not always helpful.